docs/tools/terraform/reference/cookbook/kafka-debezium-postgres-source.rst

```diff
@@ -0,0 +1,197 @@
+Apache Kafka® with Debezium source connector for PostgreSQL®
+==========================================================
+
+The `Aiven Terraform Provider <https://registry.terraform.io/providers/aiven/aiven/latest/docs>`_ is a great choice for provisioning an Aiven for Apache Kafka® cluster with Kafka Connect enabled and the `Debezium source connector for PostgreSQL® <https://developer.aiven.io/docs/products/kafka/kafka-connect/howto/debezium-source-connector-pg.html>`_ configured.
+
+Let's check out the following diagram to understand the setup.
```

Comment:
I would expand on this further. This is not only a recipe for Debezium but, on a higher level, how to integrate Kafka and Kafka Connect with services that may not be deployed in the same cloud.